cranelift/assembler-x64/meta/src/generate/format.rs

```diff
@@ -88,21 +88,20 @@ impl dsl::Format {
             [FixedReg(dst), Imm(_)] => {
                 // TODO: don't emit REX byte here.
                 fmtln!(f, "let {dst} = {};", dst.generate_fixed_reg().unwrap());
-                fmtln!(f, "let digit = 0x{:x};", rex.digit);
+                fmtln!(f, "let digit = 0;"); // No digit for this pattern.
```

Comment:
Should we `assert_eq!(rex.digit, None)` here?

Reply:
Yeah, good point.